docs/mine/storage-mining.md

```diff
@@ -122,3 +122,107 @@ Example:
 ```
 $ lotus-miner actor set-addrs /ip4/123.123.73.123/tcp/12345 /ip4/223.223.83.223/tcp/23456
 ```
+
+
+## Configuration
```

Comment:
Can this be PRed in `feat/docs-reorg`? There should be a placeholder guide (currently empty) called `miner-configuration.md`.

Reply:
Taken care of.